installing/installing_aws/installing-aws-outposts-remote-workers.adoc

```diff
@@ -12,34 +12,55 @@ This can be achieved by customizing the default AWS installation and performing
 * You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
 * You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster.
-* You are familiar with the instance types are supported in the AWS Outpost instance you use. This can be validated with link:https://docs.aws.amazon.com/cli/latest/reference/outposts/get-outpost-instance-types.html[get-outpost-instance-types AWS CLI command]
-* You are familiar with the AWS Outpost instance details, such as OutpostArn and AvailabilityZone. This can be validated with link:https://docs.aws.amazon.com/cli/latest/reference/outposts/list-outposts.html[list-outposts AWS CLI command]
 +
 [IMPORTANT]
 ====
-Since the cluster uses the provided AWS credentials to create AWS resources for its entire life cycle, the credentials must be key-based and long-term. So, If you have an AWS profile stored on your computer, it must not use a temporary session token, generated while using a multi-factor authentication device. For more information about generating the appropriate keys, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users] in the AWS documentation. You may supply the keys when you run the installation program.
+If you have an AWS profile stored on your computer, it must not use a temporary session token that you generated while using a multi-factor authentication device. The cluster continues to use your current AWS credentials to create AWS resources for the entire life of the cluster, so you must use long-term credentials. To generate appropriate keys, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users] in the AWS documentation. You can supply the keys when you run the installation program.
+//For some reason the Outposts version of this note was reworded from the existing content in VPC and moved to attach to the wrong bullet item. Reverted to VPC wording and put back in the right place.
 ====
-* You have access to an existing Amazon Virtual Private Cloud (VPC) in Amazon Web Services (AWS). See the section "About using a custom VPC" for more information.
 * If a firewall is used, it was xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured to allow the sites] that your cluster requires access to.
 
+//Outposts-specific items pulled down below this line and reworded:
+* You are familiar with the instance types are supported in the AWS Outposts instance you want to use.
+You can validate this with the link:https://docs.aws.amazon.com/cli/latest/reference/outposts/get-outpost-instance-types.html[`get-outpost-instance-types`` AWS CLI command].
+* u are familiar with the AWS Outposts instance details, such as `OutpostArn` and `AvailabilityZone`.
```

Comment:
```suggestion
* You are familiar with the AWS Outposts instance details, such as `OutpostArn` and `AvailabilityZone`.
```
NO U :anger:

Reply:
Also idk what `OutpostArn` and `AvailabilityZone` are, but they probably need to have a noun following them or be written in plain text instead of the param name.